src/deepcell/data_splitter.py

```diff
@@ -136,11 +178,10 @@ def get_cross_val_split_idxs(
         -------
         yields Tuple of np.ndarray for train and test indices
         """
-        n = len(model_inputs)
-        y = RoiDataset.get_numeric_labels(model_inputs=model_inputs)
         skf = StratifiedKFold(n_splits=n_splits, shuffle=shuffle,
                               random_state=seed)
-        for train_index, test_index in skf.split(np.zeros(n), y):
+        for train_index, test_index in skf.split(self._exp_labels,
```

Comment:
I think this will mean we have to write two `sample_data` methods. One for indexed by experiment and one indexed by ROI (aka the previous logic).

Reply:
I'm not sure I follow that. If the logic stays as is, why would you need 2 `sample_data` methods?

Reply:
Because in one case the index is on the experiment and in the other the index is on the ROI. I'll have to at least put in a bit of code to convert the indices so we can reuse the _sample method for the train/test split.

Reply:
There should only be 1 sample method I think. Just pass in indices at the ROI level rather than at the experiment level.